### PR TITLE
toph and hadley added as authors

### PR DESCRIPTION
**Pull request by aronatkins**

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -5,6 +5,8 @@ Title: Deploy Docs, Apps, and APIs to Posit Connect, shinyapps.io, and
 Version: 0.8.29-1
 Authors@R: c(
     person("Aron", "Atkins", , "aron@posit.co", role = c("aut", "cre")),
+    person("Toph", "Allen", , "toph@posit.co", role = "aut"),
+    person("Hadley", "Wickham", , "hadley@posit.co", role = "aut"),
```
Comment (hadley) [inline DESCRIPTION:9]: FWIW we normally only supply the email address for the maintainer, since that's the only place where it's required.

**Review by hadley — APPROVED** [review]
Thanks!

File: DESCRIPTION (hunk 1 above)
Comment (aronatkins) [reply DESCRIPTION:9]: Got it. I was following the existing local style. Jonathan left both our email addresses when he added me as maintainer; I'll strip the others.

Comment (aronatkins) [reply DESCRIPTION:9]: https://github.com/rstudio/rsconnect/commit/7feafbf1fd92e7511aefc3b6b870faa1a4d22c03, for reference.

**Review by toph-allen — APPROVED** [review]

**Pull request merged**